scripts/kubestellar

```diff
@@ -117,12 +82,13 @@ function ensure_wmws() {
             if [ "$i" == "" ] ; then
                 continue
             fi
-            echo "Ensuring WMW: $i"
-            ensure_wmw "$i"
-            kubectl ws use $initial_ws &> /dev/null # go back to the initial state
+            wmw=${i#"root:"}
+            echo "Ensuring WMW: $wmw"
+            kubectl ws root &> /dev/null
+            ${bindir}/kubectl-kubestellar-ensure-wmw $wmw $xflag --with-kube true &> /dev/null
```

Comment:
The `&> /dev/null` makes the `$xflag` moot, right?
Does not have to be addressed in this PR.